internal/metrics/listener.go

```diff
@@ -2,14 +2,17 @@ package metrics
 
 import (
 	"fmt"
```

Comment:
Is it possible to address the linting errors?

Reply:
I thought I had it setup correctly, I've added 'go imports' to the file watcher so it won't happen again I think.